src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs

```diff
@@ -183,19 +181,26 @@ private unsafe IPEndPoint Start(QuicListenerOptions options)
 
             Internals.SocketAddress address = IPEndPointExtensions.Serialize(listenEndPoint);
 
-            uint status;
-
             Debug.Assert(_stateHandle.IsAllocated);
 
             MemoryHandle[]? handles = null;
-            QuicBuffer[]? buffers = null;
+            // TODO: should they be pinned? The content is pinned, but the array holding the individual buffers isn't.
+            QUIC_BUFFER[]? buffers = null;
```

Comment:
The buffer doesn't need to live longer than the call to `ListenerStart`.

Reply:
Same as a comment above, I'll fix the pinning.